pegasus/sites.v3/hourofcode.com/i18n/public/bn/tos.md

```diff
@@ -1,6 +1,6 @@
 ---
 
-শিরোনামঃ সেবার শর্ত সমূহ
+title: সেবার শর্ত সমূহ
```

Comment:
I don't think the TOS should even be in the translation pipeline. @sukykang is it intentional we're translating TOS? Previously we decided not to because of legal/culpability reasons.

Reply:
Ah, ok nevermind. For another day!